docs/output.md

```diff
@@ -111,7 +111,7 @@ The results from `--steps impute` will have the following directory structure:
 ```tree
 ├── batch
 ├── csv
-├── glimpse1/glimpse2/quilt/stitch
+├── glimpse1/glimpse2/quilt/stitch/beagle5
```

Comment:
The first time I looked at this, I thought why is everything nested? But on a second look, I realized they were actually options. It might help to indicate that more clearly, using a `|` separator comes to mind.
Still, it’s not a big issue.